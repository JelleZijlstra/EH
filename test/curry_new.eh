#!/usr/bin/ehi

foo a = b => a + b

printvar(foo 1)
printvar(foo 1 2)
