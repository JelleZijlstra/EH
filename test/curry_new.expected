%%stdout%%
@function <user>: b
@int 3
%%stderr%%
